src/Docker.PowerShell/Cmdlets/ImportContainerImage.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Threading.Tasks;
+using System.Management.Automation;
+using Docker.DotNet.Models;
+using System.IO;
+using Newtonsoft.Json;
+using System.Text;
+using System.Linq;
+using Docker.PowerShell.Objects;
+
+namespace Docker.PowerShell.Cmdlets
+{
+    [Cmdlet(VerbsData.Import, "ContainerImage",
+            DefaultParameterSetName = CommonParameterSetNames.Default)]
+    [Alias("Load-ContainerImage")]
+    [OutputType(typeof(ImagesListResponse))]
+    public class ImportContainerImage : DkrCmdlet
+    {
+        private const string _loadedImage = "Loaded image: ";
+
+        #region Parameters
+
+        [Parameter(Position = 0,
+            Mandatory = true)]
+        [ValidateNotNullOrEmpty]
+        public string[] FilePath { get; set; }
+
+        #endregion
+
+        #region Overrides
+
+        protected override async Task ProcessRecordAsync()
+        {
+            foreach (var item in FilePath)
+            {
+                var filePath = System.IO.Path.Combine(SessionState.Path.CurrentFileSystemLocation.Path, item);
```

Comment:
This works, because if the second argument to Combine is a rooted path, it just returns that path instead of combining.